app/models/form/setup/questions/location_id.rb

```diff
@@ -5,6 +5,7 @@ def initialize(_id, hsh, page)
     @header = "Which location is this log for?"
     @hint_text = ""
     @type = "radio"
+    @extra_check_answer_value = "location_admin_district"
```

Comment:
Given that the value of this isn't actually used for anything and all the logic is in the method now anyway do we need this at all? 

i.e. could we just have

```ruby
# app/models/form/question.rb

def get_extra_check_answer_value(case_log)
  return nil
end


# app/models/form/setup/questions/location_id.rb

def get_extra_check_answer_value(case_log)
  case_log.form.get_question("la", nil).label_from_value(case_log.la)
end
```

Reply:
With the way it is implemented now, I guess not.